package.json

```diff
@@ -22,5 +22,6 @@
     "eslint-config-next": "14.0.1",
     "postcss": "^8",
     "tailwindcss": "^3.3.0"
-  }
+  },
+  "type": "module"
```

Comment:
Did this change still need to be implemented?

Reply:
Oh no, I thought I removed that. This can be left out